projects/plugins/jetpack/extensions/blocks/conversation/utils.js

```diff
@@ -59,17 +59,20 @@ export const TRANSCRIPT_MAX_FILE_SIZE = 100000;
  * ----------------------------
  * Serices: otter.ai, ...
  */
-const otterFormatRegExp = /(.*[^\s])\s{1,}(\d{1,2}(?::\d{1,2}?)+)\s+\n([\s\S]*?(?=\n{2}|$))/gm;
-const otterFormatTestRegExp = /(.*[^\s])\s{1,}(\d{1,2}(?::\d{1,2}?)+)\s+\n([\s\S]*?(?=\n{2}|$))/g;
+
+const otterRegExp = /(.*[^\s])\s{1,}(\d{1,2}(?::\d{1,2}?)+)\s+\n([\s\S]*?(?=\n{2}|$))/;
+const otterFormatRegExp = new RegExp( otterRegExp, 'gm' );
+const otterFormatTestRegExp = new RegExp( otterRegExp, 'g' );
 
 /*
  * ----------------------------
  * <speaker>: [<timestamp>] <content>
  * ----------------------------
  * Serices: sonix.ai, ...
  */
-const sonixFormatRegExp = /(?:(.*[^\s]):\s*)?\[([\d{1,2}:]*)]\s([\s\S]*?(?=\n{1,2}|$))/gm;
-const sonixFormatTestRegExp = /(.*[^\s]):\s*\[([\d{1,2}:]*)]\s([\s\S]*?(?=\n{1,2}|$))/g;
+const sonixRegExp = /(?:(.*[^\s]):\s*)?\[([\d{1,2}:]*)]\s([\s\S]*?(?=\n{1,2}|$))/gm;
+const sonixFormatRegExp = new RegExp( sonixRegExp, 'gm' );
+const sonixFormatTestRegExp = new RegExp( sonixRegExp, 'gm' );
```

Comment:
I think this one could be just `g` and you could remove the modifier from `sonixRegExp` but that doesn't matter

Reply:
👍 correct https://github.com/Automattic/jetpack/pull/19005/commits/719eb3af3ca0e199f5362bc14eff2620b065745b